### DISCUSSION TITLE
New mamba and Python versions

### DISCUSSION THREAD

**Discussion by valeriupredoi** [ESMValGroup/ESMValCore#1781] [Announcements]
Let's take a moment and observe the first mamba version-1.0.0 uploaded to conda-forge just a couple minutes ago https://anaconda.org/conda-forge/mamba - this goes to show the mamba deva have reached a good level of trust in their package to release the stable 1.0.0 (am currently testing it, no worries) :grin: Also, Python=3.11 is out and about and we should make sure we use it! :beer:

**Comment by zklaus** [comment]
Imho no need to be hasty with Python 3.11. Check the status of Conda Forge's migration [here](https://conda-forge.org/status/#python311). At time of writing, 1057 packages have Python 3.11 versions with 1466 packages still on the way...

**Comment by valeriupredoi** [comment]
Heads up mamba One works well :grin: @zklaus very good point! I am always on the safe side and thinking we should wait for our env to migrate to the newer Python, just something to keep in mind. I think we had some issues with 3.10 a year or so ago exactly for this reason :beer:
